doc/modules/covariance.rst

```diff
@@ -105,6 +105,23 @@ a sample with the :meth:`ledoit_wolf` function of the
 `sklearn.covariance` package, or it can be otherwise obtained by
 fitting a :class:`LedoitWolf` object to the same sample.
 
+It is important to note that when the number of samples is much larger than
+the number of features, one would expect that no shrinkage would be necessary.
+The intuition behind this is that if the population covariance is full rank,
+when the number of sample grows, the sample covariance will also become
+positive definite.  As a result, no shrinkage would necessary
+and the method should automatically do this.
+
+However, this is not the case in the LW procedure when the
+population covariance is a multiple of the identity matrix.  While at
+first this might sound like an issue, it easy to see why this is not
+the case.  When the population covariance is a multiple of the
+identity, the LW shrinkage estimate becomes close or equal to 1.
+This indicates that the optimal estimate of the covariance matrix in
+the LW sense of the is multiple of the identity.  Since the population
```

Comment:
There is a problem with this sentence. Either part of it is missing, or there is a "of the" that needs to be deleted.